src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -344,15 +349,15 @@ public Http2ReadStream(Http2Stream http2Stream)
 
                 protected override void Dispose(bool disposing)
                 {
-                    if (Interlocked.Exchange(ref _disposed, 1) != 0)
+                    Http2Stream stream = Interlocked.Exchange(ref _http2Stream, null);
```

Comment:
Naming consistency: let's name this variable "http2Stream"

Reply:
Ok